src/main/target/AOCODARCF4V3/target.h

```diff
@@ -0,0 +1,169 @@
+
+#define USE_BARO
+#define BARO_I2C_BUS                    DEFAULT_I2C_BUS
+
+#define USE_BARO_BMP280
+#define USE_BARO_MS5611
+#define USE_BARO_DPS310
+
+//*********** Magnetometer / Compass *************
+#define USE_MAG
+#define MAG_I2C_BUS                     DEFAULT_I2C_BUS
+#define USE_MAG_ALL
+
+// ******* SERIAL ********
+#define USE_VCP
+#define VBUS_SENSING_PIN        PB12
+#define VBUS_SENSING_ENABLED
+
+#define USE_UART1
+#define UART1_TX_PIN                    PA9
+#define UART1_RX_PIN                    PA10
+
+#define USE_UART2
+#define UART2_TX_PIN                    PA2
+#define UART2_RX_PIN                    PA3
+
+#define USE_UART3
+#define UART3_TX_PIN                    PC11
+#define UART3_RX_PIN                    PC10
+
+#define USE_UART4
+#define UART4_TX_PIN                    PA0
+#define UART4_RX_PIN                    PA1
+
+#define USE_UART5
+#define UART5_TX_PIN                    PC12
+#define UART5_RX_PIN                    PD2
+
+
+#define SERIAL_PORT_COUNT              6
+
+
+// ******* SPI ********
+#define USE_SPI
+
+#define USE_SPI_DEVICE_1
+#define SPI1_NSS_PIN                    PA4
+#define SPI1_SCK_PIN                    PA5
+#define SPI1_MISO_PIN                   PA6
+#define SPI1_MOSI_PIN                   PA7
+
+#define USE_SPI_DEVICE_2
+#define SPI2_NSS_PIN                    PA13
+#define SPI2_SCK_PIN                    PB13
+#define SPI2_MISO_PIN                   PB14
+#define SPI2_MOSI_PIN                   PB15
+
+#define USE_SPI_DEVICE_3
+#define SPI3_NSS_PIN                    PC0
+#define SPI3_SCK_PIN                    PB3
+#define SPI3_MISO_PIN                   PB4
+#define SPI3_MOSI_PIN                   PB5
+
+// ******* ADC ********
+#define USE_ADC
+#define ADC_CHANNEL_1_PIN               PC1
+#define ADC_CHANNEL_2_PIN               PC2
+#define ADC_CHANNEL_3_PIN               PC3
+
+#define VBAT_ADC_CHANNEL                ADC_CHN_2
+#define RSSI_ADC_CHANNEL                ADC_CHN_3
+#define CURRENT_METER_ADC_CHANNEL       ADC_CHN_1
+
+#define VBAT_SCALE_DEFAULT              1100
+
+// ******* OSD ********
+#define USE_OSD
+#define USE_MAX7456
+#define MAX7456_SPI_BUS                 BUS_SPI2
+#define MAX7456_CS_PIN                  PA13
+
+//******* FLASH ********
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+#define M25P16_SPI_BUS                  BUS_SPI3
+#define M25P16_CS_PIN                   PC0
+
+#define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
+
+// *************** PINIO ***************************
+#define USE_PINIO
+#define USE_PINIOBOX
+#define PINIO1_PIN                  PC5  // VTX power switcher
+#define PINIO2_PIN                  PA14  //
+
+//************ LEDSTRIP *****************
+#define USE_LED_STRIP
+#define WS2811_PIN                      PB1
+
+// ******* FEATURES ********
+#define DEFAULT_RX_TYPE                 RX_TYPE_SERIAL
+#define SERIALRX_UART                   SERIAL_PORT_USART2
+#define SERIALRX_PROVIDER               SERIALRX_SBUS
```

Comment:
We use SERIALRX_CRSF for default settings. That's used for our PNP 7 inch copters.